plugins/GrapesJsBuilderBundle/Controller/FileManagerController.php

```diff
@@ -37,4 +37,17 @@ public function deleteAction()
 
         return $this->sendJsonResponse(['success'=> true]);
     }
+
+    /**
+     * @return JsonResponse
+     */
+    public function assetsAction()
```

Comment:
```suggestion

    public function assetsAction(): JsonResponse
```

Reply:
Please create a functional test that will hit this endpoint and check that some pre-installed asset is present in the response.